dockerfile

```diff
@@ -1,20 +1,15 @@
-FROM golang:1.19-alpine
-
-RUN apk add --no-cache git
-
-# Set the Current Working Directory inside the container
```

Comment:
In general, I don't like straightforward comments, they don't have much added value and just spam the code. In this instance, the WORKDIR command is very clear and doesn't need any more explanation. Although, I accept these when they appear in a tutorial type code to help someone new to the language/code.